src/App.tsx

```diff
@@ -1,15 +1,25 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const variablesMovies = moviesFromServer.filter(movie => (
+    movie.title.toLowerCase().includes(query.toLowerCase().trim())
+    || movie.description.toLowerCase().includes(query.toLowerCase().trim())
```

Comment:
duplicate. create helper function and use it for title as well as for description